### ISSUE TITLE
Post-Release-List

### ISSUE THREAD

**Issue by lgrahl** [saltyrtc/saltyrtc-meta#114]
- [ ] Update the readme to refer to `master` and tagged versions
- [ ] Update saltyrtc.org

**Comment by dbrgn** [comment]
Last blocker as far as I can see: #118.

**Comment by lgrahl** [comment]
#118 is resolved for me now. Last thing: #126

**Comment by lgrahl** [comment]
What's missing @dbrgn. Can we finally release?

**Comment by dbrgn** [comment]
Hm, there was something I wanted to take a look at before we release. I'll take a look on Monday, maybe I can fix that then.

**Comment by lgrahl** [comment]
I think it's time. :smile_cat:

**Comment by lgrahl** [comment]
Postponed until we're sitting at the same table.

**Comment by dbrgn** [comment]
Yeah, I think it's actually good we waited, since the relayed data task brought some changes (like the 'disconnected' message).
